destination/destination_integration_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"testing"
+
+	"github.com/conduitio-labs/conduit-connector-nats-pubsub/config"
+	"github.com/conduitio-labs/conduit-connector-nats-pubsub/test"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+)
+
+func TestDestination_Open(t *testing.T) {
+	t.Parallel()
+
+	t.Run("success, pubsub", func(t *testing.T) {
```

Comment:
Maybe run all of those below as individual tests test functions?

Reply:
Yeah, okay. I split those subtests into separate test functions.